Makefile

```diff
@@ -382,7 +384,7 @@ endif
 WARNINGS = all error strict-prototypes missing-prototypes nested-externs \
 	missing-declarations undef pointer-arith no-nonnull declaration-after-statement
 
-CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding \
+CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding -fms-extensions \
```

Comment:
You can rewrite this commit and force push to this PR, or open a new one, whichever is easier.

Whilst modifying the commit though it would be nice if you tried to conform with our normal commit styling though (do a git log through our commits), which is a short summary line, a blank line, then any necessary paragraphs giving more detail.